scripts/test_critical_features_simple.py

```
@@ -205,7 +204,7 @@
             mock_connect.return_value = mock_ws
 
             # Test connection
-            watcher = GengoWatcher(config, logger, state)
+            watcher = GengoWatcher(config, state, logger)
```

Ruff (0.14.5)

Local variable `watcher` is assigned to but never used

Remove assignment to unused variable `watcher`

(F841)